tests/rptest/tests/end_to_end.py

```diff
@@ -61,12 +61,14 @@ def __init__(self, test_context, extra_rp_conf=None):
         self.redpanda = None
         self.topic = None
 
-    def start_redpanda(self, num_nodes=1):
+    def start_redpanda(self, num_nodes=1, extra_rp_conf=None):
+        if not extra_rp_conf:
+            extra_rp_conf = self._extra_rp_conf
```

Comment:
merge or replace? if you really want to replace all the config options, maybe rename?